internal/sql/migrations.go

```diff
@@ -193,6 +193,22 @@ var (
 					return performMigration(ctx, tx, migrationsFs, dbIdentifier, "00013_coreutils_wallet", log)
 				},
 			},
+			{
+				ID: "00014_hosts_resolvedaddresses",
```

Comment:
For migrations we usually create a migration sql file for both `sqlite` and `mysql`. This PR is also missing the change to the `schema.sql` file (which is why some tests fail currently).

Reply:
Also keep in mind that the migrations should be manually tested. e.g. by spinning up a testnet node for each SQLite and MySQL.